src/lib/SnackBar/index.ts

```diff
@@ -0,0 +1,98 @@
+import './styles.css';
+
+const DEFAULT_TIMEOUT = 2750;
+
+export interface SnackBarOptions {
+  message: string;
+  timeout?: number;
+  actionText?: string;
+  actionHandler?: () => boolean | null;
+}
+
+class SnackBar {
+  on?: () => void;
```

Comment:
Should this be optional given it has a default value?

Reply:
I'm not sure what the `on?` was from, that seemed like a typo. `onhide` is an optional hook, it is only invoked if it exists.